Shared/Models/Eatery.swift

```diff
@@ -255,6 +255,25 @@ extension Eatery {
         }
     }
 
+    var hasFavorite: Bool {
+        let events = self.eventsByName(onDayOf: Date())
+        for event in events {
+            for (_, items) in event.value.menu.data {
+                if items.contains(where: { item in
+                    DefaultsKeys.isFavoriteFood(item.name)
+                }) {
+                    return true
+                }
+            }
+        }
+        return false
+    }
+
+}
+
+extension UIImage {
```

Comment:
White space before/after the curly brackets of extension, class declarations, and struct declarations.